src/components/ListHeader.vue

```diff
@@ -130,6 +131,11 @@ export default {
 </script>
 
 <style>
+.file-picker-header {
+  position: sticky;
+  background-color: var(--brand-primary);
```

Comment:
Color vars have changed, see `src/assets/tokens/ods.scss` in ODS. Guess that specific one needs to be `--oc-color-swatch-primary-default`